src/components/Navbar/MenuItems.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import * as FaIcons from "react-icons/fa";
```

Comment:
The `package.json` file with `react-icons` as a dependency hasn't been added in any commits. Second, what is the point of importing the `whole` library into the file if you are using only those icons you need from it (`import { A, B, C } from 'react-icons/fa'`) and not the whole library (`... * as FaIcons from ...`).